AirportsLatLongConverter.py

```diff
@@ -0,0 +1,20 @@
+import airportsdata
+
+#airports = airportsdata.load('IATA')
+
+# Should we add a name to abbreviation converter?
+# How will users input their airport?
+
+# Get lat and lon
+def get_lat_and_lon(input_airport):

```

Comment:
documentation string after this function definition explaining what it expects as input and what kind of return it gives back (e.g., return value will be a two-element list of float values representing latitude and longitude)